### PR TITLE
Improve Traefik Fault Tolerance

### PR DESCRIPTION
**Pull request by shaneutt**
Increases the number of default traefik pods deployed to help reduce ingress downtime from individual node failures.

### CODE REVIEW

**Comment by s12chung** [comment]
can you write why we need more replicas? a card or something?

File: templates/traefik.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ spec:
     version: 1.68.4
     values: |
       ---
+      replicas: 3
```
Comment (hectorj2f) [inline templates/traefik.yaml]: two replicas should be enough. Did we face any specific issue to increase the replicas ?

Comment (shaneutt) [reply templates/traefik.yaml]: hang on, this is `wip` I'm looking at the potential autoscaling feature and will come back to this shortly.

**Comment by shaneutt** [comment]
Closing for the moment to get it off the queue, will re-open later

File: templates/traefik.yaml (hunk 1 above)
Comment (hectorj2f) [reply templates/traefik.yaml]: Alright :), roger that.

**Comment by shaneutt** [comment]
@hectorj2f ready for re-review, skipping autoscale for the moment and focusing on a simple and quick turnaround for the moment.

**Pull request merged**